presto-hive/src/main/java/com/facebook/presto/hive/metastore/thrift/StaticMetastoreConfig.java

```diff
@@ -63,4 +67,30 @@ public StaticMetastoreConfig setMetastoreUsername(String metastoreUsername)
         this.metastoreUsername = metastoreUsername;
         return this;
     }
+
+    public Duration getMaxMetastoreBackoff()
+    {
+        return maxMetastoreBackoff;
+    }
+
+    @Config("hive.metastore.max-backoff")
+    @ConfigDescription("max backoff to retry the failed metastore")
+    public StaticMetastoreConfig setMaxMetastoreBackoff(Duration maxMetastoreBackoff)
+    {
+        this.maxMetastoreBackoff = maxMetastoreBackoff;
+        return this;
+    }
+
+    public Duration getMinMetastoreBackoff()
+    {
+        return minMetastoreBackoff;
+    }
+
+    @Config("hive.metastore.min-backoff")
+    @ConfigDescription("min backoff to retry the failed metastore")
```

Comment:
this doesn't look accurate. We can retry failed metastore (almost) immediately, if all other failed as well